zerver/lib/notifications.py

```diff
@@ -302,8 +298,10 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
         else:
             huddle_display_name = u"%s, and %s others" % (', '.join(other_recipients[:2]), len(other_recipients) - 2)
             context.update({'huddle_display_name': huddle_display_name})
+        reply_to_name = huddle_display_name
     elif (missed_messages[0].recipient.type == Recipient.PERSONAL):
         context.update({'private_message': True})
+        reply_to_name = cast(Text, missed_messages[0].sender.full_name)
```

Comment:
I think let's just use something like https://docs.python.org/2/library/email.header.html in send_email and maybe call it a day. 

What does this cast for?